installer/app/src/views/release-channel-selector.js.jsx

```diff
@@ -0,0 +1,82 @@
+import Dispatcher from '../dispatcher';
+import Config from '../config';
+import ExternalLink from './external-link';
+
+var ReleaseChannelSelector = React.createClass({
+	render: function () {
+		var clusterState = this.props.state;
+		var releaseChannels = Config.release_channels || [];
+		var selectedReleaseChannel = (clusterState.releaseChannel ? (releaseChannels.find(function (ch) {
+			return ch.name === clusterState.releaseChannel;
+		})) : null) || null;
+		var selectedReleaseVersion = clusterState.releaseVersion;
+
+		var ec2Versions = Config.ec2_versions;
+		if (clusterState.selectedCloud === 'aws') {
```

Comment:
It would be nice to have an explanation in the UI why you can't select `nightly` for AWS (I was surprised, but it makes sense after reading your PR comment).